lms/templates/video.html

```diff
@@ -48,7 +48,7 @@ <h4 class="hd hd-4 video-hls-error is-hidden">
 
     <div class="focus_grabber last"></div>
 
-    % if (download_video_link or track or handout or branding_info or public_sharing_enabled) and not hide_downloads:
+    % if ((download_video_link or track or handout or branding_info or public_sharing_enabled) and not hide_downloads) or transcript_feedback_enabled:
```

Comment:
wouldn't it be better if we just abstract the `transcript_feedback_enabled` block from outside these conditionals?